socorro/processor/cache_manager.py

```diff
@@ -449,14 +479,24 @@ def _event_generator(self, nonblocking=False):
                         len(self.lru),
                         f"{self.total_size:,d}",
                     )
-                    # Emit a usage metric, but debounce it so it only gets emitted at
-                    # most once per second
-                    now = datetime.datetime.now()
-                    if now > (last_usage_update + datetime.timedelta(seconds=1)):
-                        METRICS.gauge("usage", value=self.total_size)
-                        last_usage_update = now
                     processed_events = False
 
+                # Emit usage metric, but debounce it so it only gets emitted at most
+                # once per second
+                now = time.time()
+                if now > next_heartbeat:
+                    if is_verbose:
+                        METRICS.gauge("usage", value=self.total_size)
+
+                    sorted_sizes = list(sorted(self.lru.values()))
+                    if sorted_sizes:
+                        METRICS.gauge("file_sizes.median", get_index(sorted_sizes, 50))
```

Comment:
These two metrics felt like the ones I want to see over time to figure out if the size of files in the disk cache (and thus the read/write work to use them) was increasing over time as the processor aged.